src/Plugin.php

```diff
@@ -125,12 +125,29 @@ public static function renderTemplate(array $template_subpathnames, array $varia
   }
 
   /**
-   * Checks if wp-rocket plugin is active and images lazyload option is set.
+   * Checks if images are being lazy loaded.
    *
    * @return bool
    */
   public static function isLazyLoadActive() {
-    return is_plugin_active('wp-rocket/wp-rocket.php') && get_rocket_option('lazyload');
+    return apply_filters('gallerya_lazyload_is_active', FALSE);
```

Comment:
Before this PR, gallerya automatically detected that wp-rocket had lazy-loading enabled. Now it doesn't anymore. The value is always FALSE, unless filtered manually in a custom plugin. Why are we degrading the experience?

Reply:
@sun I finally went with your last suggested approach. Much simpler, general and also harmless.